src/schema.proto

```diff
@@ -1,7 +1,11 @@
 syntax = "proto3";
-package codecprotobuf;
 
-message AnyType {
-  required string type = 1;
+// TODO(burdon): Rename dxos.any (and entire package?)
+// TODO(burdon): Why not import "google/protobuf/any.proto"?
```

Comment:
Probably because the first version wasn't using protobufjs and we end up with this solution. Remove todo, open an issue.